docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -697,6 +697,12 @@ and a switch section for each switch (managed by PacketFence) including:
 NOTE: `switches.conf` is loaded at startup. A reload is required when changes 
 are manually made to this file `/usr/local/pf/bin/pfcmd configreload`.
 
+Import from CSV
+^^^^^^^^^^^^^^^
+From PacketFence 7.2 and onwards, you have the possibility to import switches from a CSV.
+
+This tool allow you to import switches form a CSV file. The following information in this particular order are necessary to use this tool: "description, IP or MAC, switch group". Each line which has all three will create an entry in the switch configuration file. You MUST create switches group before.
```

Comment:
This tool? If it's fro mthe Web GUI, it's not a tool - a tool is generally run from the command line. Please rephrase.